### DISCUSSION TITLE
https post certificate verification error Pico W

### DISCUSSION THREAD

**Discussion by hgprojekt** [micropython/micropython#13449] [RP2040 / Pico]
I'm new in the forum so hello everyone.

I'm trying to use the requests library to send data to the server via https protocol.
`r = requests.post(url, headers=headers, data=json.dumps(data), verify='server.pem')`

Certificate is stored locally in server.pem file.
Everything works great in Python.

In Micropython (1.22.1) there is an error:
TypeError: unexpected keyword argument 'verify'

How to add a certificate verification to a requests.post?

**Comment by sophiedegran** [comment]
Hi
In micropython certificate have to be send in DER format not PEM
Have a look to microdot tls example
